test/arrays/userAPI/arrayAPItest.chpl

```diff
@@ -57,10 +57,11 @@ proc testArrayAPI2D(lbl, X: [], sliceDom, reindexDom) {
   writeln("X is:\n", X);
   writeln();
 
-  // Test read access via tuples and varargs
-  writeln("low element is: ", X.localAccess[X.domain.alignedLow]);
+  const domForLowHigh = if X.hasSingleLocalSubdomain() then X.localSubdomain() else X.domain;
+  // Test local read access via tuples and varargs
+  writeln("low element is: ", X.localAccess[domForLowHigh.alignedLow]);
```

Comment:
Yeah, I'm just relying on:
- us being the only ones to call this arrayAPItest
- there being some indices owned by the current locale